internal/cmd/schema-registry/command_config_delete.go

```diff
@@ -0,0 +1,91 @@
+package schemaregistry
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	v1 "github.com/confluentinc/cli/internal/pkg/config/v1"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/form"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+func (c *command) newConfigDeleteCommand(cfg *v1.Config) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "delete",
+		Short: "Delete top-level or subject-level schema configuration.",
+		Args:  cobra.NoArgs,
+		RunE:  c.configDelete,
+	}
+
+	example1 := examples.Example{
+		Text: `Delete the subject-level configuration of subject "payments".`,
+		Code: "confluent schema-registry config delete --subject payments",
+	}
+	example2 := examples.Example{
+		Text: "Delete the top-level configuration.",
+		Code: "confluent schema-registry config delete",
+	}
```

Comment:
Nit: Let's swap these examples, so the example with the top-level configuration is on top